packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -92,7 +92,7 @@ const NewNetwork = decorate([
             description,
             mtu,
             name,
-            pif: pif.id,
+            pif: pif && pif.id,
```

Comment:
`pif != null && pif.id`  ?

Reply:
I don't want to pass `false`. I either want `undefined` or an ID. But I'll explicit the test.